test/Functional/Tests/GenerationTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+namespace Hostnet\Component\EntityPlugin\Functional;
+
+use Composer\Autoload\AutoloadGenerator;
+use Composer\Composer;
+use Composer\Config;
+use Composer\EventDispatcher\EventDispatcher;
+use Composer\IO\BufferIO;
+use Composer\Package\RootPackage;
+use Composer\Repository\InstalledArrayRepository;
+use Composer\Repository\RepositoryManager;
+use Hostnet\Component\EntityPlugin\Plugin;
+use RecursiveDirectoryIterator;
+use RecursiveIteratorIterator;
+use Symfony\Component\Console\Output\StreamOutput;
+
+class GenerationTest extends \PHPUnit_Framework_TestCase
```

Comment:
Missing @covers annotation? An integration test like this shouldn't cover anything.